packages/dialtone-vue2/components/split_button/split_button-omega.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <dt-button
+    :id="id"
+    v-dt-tooltip="tooltipText"
+    data-qa="dt-split-button-omega"
+    :size="size"
+    :active="active"
+    :aria-label="ariaLabel"
+    :disabled="disabled"
+    :importance="importance"
+    :kind="kind"
+    :class="`d-split-btn__omega d-split-btn__omega--${size}`"
+    @click="(e) => $emit('click', e)"
+  >
+    <template #icon>
+      <slot name="icon">
+        <dt-icon-chevron-down :size="SPLIT_BUTTON_ICON_SIZES[size]" />
```

Comment:
Im thinking we should have the `dt-icon` component here to manage the icons with the default value of `chevron-down`. 
Access through with `omegaIcon` prop.

Reply:
We're trying to make Dialtone-vue fully tree-shakeable and in order to achieve that, we need to stop using <dt-icon> as it includes all the icons in the bundle even if you're using a single icon.

With this, it'd only include the chevron icon and the use can pass whenever icon it needs, importing only the icon it needs.

Reply:
Yep we'll be doing a migration next sprint to remove all use of dt-icon in dialtone-vue.